airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/csv/S3CsvWriter.java

```diff
@@ -100,73 +78,22 @@ public S3CsvOutputFormatter(S3DestinationConfig config,
     // once it has reached it's configured part size.
     // See {@link S3DestinationConstants} for memory usage calculation.
     this.uploadManager = new StreamTransferManager(config.getBucketName(), objectKey, s3Client)
-        .numStreams(S3DestinationConstants.DEFAULT_NUM_STREAMS)
-        .queueCapacity(S3DestinationConstants.DEFAULT_QUEUE_CAPACITY)
-        .numUploadThreads(S3DestinationConstants.DEFAULT_UPLOAD_THREADS)
-        .partSize(S3DestinationConstants.DEFAULT_PART_SIZE_MD);
+        .numStreams(S3CsvConstants.DEFAULT_NUM_STREAMS)
```

Comment:
WHy don't we use the same hadoop s3 uploader here?

Reply:
If I am reading the PR correctly, it seems we are using two different ways to push data to s3. 
- ParquetWriter
- StreamTransferManager

I am just curious if it is possible to use a similar one from the Hadoop package to push the CSV one.

Reply:
Got it. The two writers output data in different data structures, and we do need them for different formats. The Parquet writer organizes data in Parquet row groups, while the stream transfer manager writes data line by line.